tiledb/bioimg/converters/ome_tiff.py

```diff
@@ -139,6 +140,118 @@ def _original_metadata(self, key: str, default: Any = None) -> Any:
+                        "name": channel.get("Name", f"Channel {idx}"),
+                        "color": get_rgba(channel["Color"])
+                        if "Color" in channel
+                        else next(color_generator),
+                    }
+                    if "EmissionWavelength" in channel:
+                        channel_metadata["emissionWavelength"] = length_converter(
+                            channel["EmissionWavelength"],
+                            channel.get("EmissionWavelengthUnit", "nm"),
+                            "nm",
+                        )
+
+                    channels.append(channel_metadata)
+
+            metadata["channels"] = channels
+
+            for dim in ["X", "Y", "Z"]:
+                if f"PhysicalSize{dim}" in image:
+                    metadata[f"physicalSize{dim}"] = length_converter(
+                        image[f"PhysicalSize{dim}"],
+                        image.get(f"PhysicalSize{dim}Unit", "μm"),
+                        "μm",
+                    )
+
+            if "TimeIncrement" in image:
+                metadata["timeIncrement"] = image["TimeIncrement"]
+                metadata["timeIncrementUnit"] = image.get("TimeIncrementUnit", "s")
+
+        else:
+            # If file is not OME we will try to extract metadata from the IFD.
+            # If you are ingesting a non-OME tiff file you may need to provide a custom metadata
+            # extraction method
+
+            page = self._tiff.pages.first
+
+            if page.photometric == tifffile.PHOTOMETRIC.RGB:
+                metadata["channels"] = [
+                    {"id": f"{idx}", "name": f"{name}", "color": next(color_generator)}
+                    for idx, name in enumerate(["red", "green", "blue"])
+                ]
+            else:
+                num_channels, color_generator = (
+                    (self._series.shape[self.axes.dims.index("C")], color_generator)
+                    if "C" in self.axes.dims
+                    else (1, iter([]))
+                )
+
+                metadata["channels"] = [
+                    {
+                        "id": f"{idx}",
+                        "name": f"Channel {idx}",
+                        "color": next(
+                            color_generator, get_rgba(4294967295)
+                        ),  # decimal representation of white
+                    }
+                    for idx in range(num_channels)
+                ]
+
+            # Aperio .svs files have a description tag with metadata split by '|' https://openslide.org/formats/aperio/
+            if self._tiff.is_svs:
+                info = {}
+                for entry in page.description.split("\n")[1].split("|"):
+                    key, value = entry.split("=")
+                    info[key] = value
+
+                if "MPP" in info:
+                    metadata["physicalSizeX"] = info.get("MPP")
+                    metadata["physicalSizeΥ"] = info.get("MPP")
```

Comment:
if the same why not `metadata["physicalSizeX"] = metadata["physicalSizeΥ"] = info.get("MPP")`?